css/index.css

```diff
@@ -0,0 +1,43 @@
+@import "./navbar.css";
+
+.container-main__content {
+  width: 90%;
+}
+
+.main__heading {
+  font-size: 4rem;
+  font-family: var(--font-heading);
+  font-weight: bolder;
+}
+
+.categories__heading {
+  font-size: 2.5rem;
+  font-weight: bolder;
+  border-bottom: 0.1rem solid var(--dark);
+}
+
+.categories__container {
+  width: 100%;
+  flex-wrap: wrap;
+  border: 0.1rem solid var(--dark);
+  overflow-y: scroll;
+  justify-content: space-evenly;
+}
+
+.categories__container .card {
+  width: 25rem;
+  margin: 1rem 1rem 1rem 1rem;
```

Comment:
use margin:1rem instead of margin:1rem 1rem 1rem1rem

Reply:
sure i'll fix that.